sources/core/Stride.Core.Mathematics/UInt4.cs

```diff
@@ -183,6 +183,12 @@ public UInt4(uint[] values)
             }
         }
 
+        /// <summary>
+        /// Casts from System.Numerics to Stride.Maths vectors
+        /// </summary>
+        /// <param name="v">Value to cast</param>
+        public static implicit operator UInt4(System.Numerics.Vector4 v) => new((uint)v.X, (uint)v.Y,(uint)v.Z,(uint)v.W);
```

Comment:
Conversion must be explicit when data is lost in the process, similar to how c# does it between the base value types

Reply:
I understand! will change it on the next commit